### PR TITLE
change the test case to use Prometheus alerts rather that openshift console

### PR DESCRIPTION
**Pull request by austincunningham**
# Description

change the test case to use Prometheus alerts rather that openshift console

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test-cases/tests/alerts/c05b-verify-alerts-are-exposed-and-no-critical-alerts-have-fired.md [hunk 1]
```diff
@@ -43,8 +43,11 @@ Testcase should not be performed on a cluster that has been used for destructive
 
 4. Check there are no currently firing alerts.
 
-   - In Openshift console, go to Monitoring -> Alerting
-   - In filter, select Source -> User
+   - Go to the middleware-monitoring prometheus namespace `oc project redhat-rhoam-middleware-monitoring-operator`
+   - Get the prometheus route `oc get routes | grep prometheus-route`
```
Comment (laurafitzgerald) [inline test-cases/tests/alerts/c05b-verify-alerts-are-exposed-and-no-critical-alerts-have-fired.md:47]:
```suggestion
   - Get the prometheus route `oc get routes -n redhat-rhoam-middleware-monitoring-operator | grep prometheus-route`
```

Comment (austincunningham) [reply test-cases/tests/alerts/c05b-verify-alerts-are-exposed-and-no-critical-alerts-have-fired.md]: :+1:

**Comment by laurafitzgerald** [comment]
/lgtm
changes look good to me.

**Pull request merged**